modules/retail/pages/retail-deploy-terminals-other.adoc

```diff
@@ -0,0 +1,205 @@
+[[retail.deploy_terminals_other]]
+= Deploy Terminals - Other Methods
+
+
+If you are not able to boot terminals from the network, you can create a live USB image and deploy terminals using a removable USB storage device.
+
+You can also bootstrap terminals across a wireless network.
+
+[IMPORTANT]
+====
+After you have registered new terminals, always check the {susemgr} {webui} to ensure your terminals have connected successfully to the branch server, and not directly to the {susemgr} Server by mistake.
+====
+
+
+
+== Deploy Terminals with a Removable USB Device
+
+If you do not want to boot terminals from the network, you can create a live USB image and deploy terminals using a removable USB storage device.
+This is useful if you cannot boot your terminals from the network, or if you do not have a local {smr} branch server providing network services.
+
+You can prepare a bootable USB device with the image and tools required to deploy a POS terminal using a remote {smr} branch server.
+POS devices booted using the USB device are assigned to the {smr} branch server that created the USB device.
+
+You can create the bootable USB device on the branch server directly, or on the {smr} Server.
+
+
+
+.Procedure: Creating a Bootable USB Device
+. On the {smr} branch server, at the command prompt, as root, create the POS image.
+You need to specify the size of the image, in megabytes.
+Ensure you allow at least 300{nbsp}MB:
++
+----
+salt-call image_sync_usb.create <usb image name> <size in MB>
+----
+. Insert the USB device into the {smr} branch server machine, and copy the image to the new location:
++
+----
+dd if=<usb image name> of=<path to usb device>
```

Comment:
to speed up, add bs=16K or something similar.

Reply:
True, in the code we use `bs=1M`.